src/main/java/org/fcrepo/client/PostBuilder.java

```diff
@@ -115,6 +115,17 @@ public PostBuilder filename(final String filename) throws FcrepoOperationFailedE
     }
 
     /**
+     * Provide a content disposition header without a filename
+     *
+     * @param disposition the disposition value
+     * @return this builder
+    **/
+    public PostBuilder disposition(final String disposition) {
```

Comment:
The only valid value for `disposition` is: `attachment *( ";" disposition-parm )` 
see: https://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html#sec19.5.1

..which means that a user-provided value that is not either `attachment` or `attachment; filename=...` would not be valid.

Since @birkland's [use case](https://jira.duraspace.org/browse/FCREPO-2499) uses `Content-Disposition: attachment`, maybe it makes sense to allow a `null` in the [existing method](https://github.com/fcrepo4-exts/fcrepo-java-client/blob/9b7121dea07f5394dbaed24bd8823a225674e823/src/main/java/org/fcrepo/client/PostBuilder.java#L103) in which case the `; filename=...` can be left off.